velox/connectors/hive/storage_adapters/s3fs/S3FileSystem.cpp

```diff
@@ -190,7 +189,8 @@ class S3ReadFile final : public ReadFile {
   int64_t length_ = -1;
 };
 
-Aws::Utils::Logging::LogLevel inferS3LogLevel(std::string level) {
+Aws::Utils::Logging::LogLevel inferS3LogLevel(std::string_view logLevel) {
+  std::string level = std::string(logLevel);
   // Convert to upper case.
   std::transform(
```

Comment:
why can't you do this with the string_view directly?

Reply:
Because we are converting to upper below.